pkg/controller/certificate-shim/sync.go

```diff
@@ -44,37 +47,46 @@ const (
 	reasonDeleteCertificate = "DeleteCertificate"
 )
 
-var ingressGVK = networkingv1beta1.SchemeGroupVersion.WithKind("Ingress")
+var ingressGVK = networkingv1beta1.SchemeGroupVersion.WithKind("IngressOrGateway")
```

Comment:
question: does the kind `IngressOrGateway` exist? 😅

Reply:
Haha, that is an overzealous string replace that happened when I added a unit test.